lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
odd...according to this:
http://eslint.org/docs/rules/strict
at 2 means never to use it @e2tha-e